chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -10,36 +10,30 @@ UPGRADEDIR=/var/lib/crowbar/upgrade
 mkdir -p "`dirname "$LOGFILE"`"
 exec >>"$LOGFILE" 2>&1
 
+hostname=$1
+if [[ "$2" = "delete-ns" ]]; then
+    delete_ns="--ssh-delete-namespace"
+else
+    delete_ns=""
+fi
+
 echo "Executing $BASH_SOURCE"
 
 set -x
 
 mkdir -p $UPGRADEDIR
 
-if [[ -f $UPGRADEDIR/crowbar-router-migration-ok ]] ; then
```

Comment:
The scripts are executed in background (over ssh) and the caller is checking for presence of these files to realize how did they end.